test/tests/functional/pbs_compress.py

```diff
@@ -38,11 +38,11 @@
 # subject to Altair's trademark licensing policies.
 
 
+from tests.functional import *
 import os
-from ptl.utils.pbs_testsuite import *
 
 
-class classcomp(PBSTestSuite):
+class TestCompression(TestFunctional):
```

Comment:
Multiple thoughts on this test suite:
1. If we are not checking log to see whether it has compressed or not then why we need this test?
2. this is something tpp related test and we have one suite for it (IIRC, its TestTPP), can we move this test in that suite?

Reply:
Yeah, I agree the purpose of the test was to match the log message use to appear after calling tpp_inflate and  tpp_deflate, now the log message is not present in the code so it is ok to remove this test.

Reply:
thanks @riyazhakki. 
@bhroam & @hirenvadalia please let me know if you are fine with removing this test.

Reply:
I am ok with removal of this suite.

Reply:
I'm fine with the removal of the test too.  It sounds like it is no longer needed.